### DISCUSSION TITLE
Организация кода | Angular с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/angdev.ru#24] [Announcements]
# Организация кода | Angular с примерами кода

Для того чтобы использование библиотеки Angular NgRx стало оправданным и максимально эффективным, необходимо правильно организовать и структурировать код, формирующий хранилище (Store) приложения

https://angdev.ru/ngrx/code-structure/

**Comment by zbralex** [comment]
вот бы еще этот пример на stackblitz "потрогать"...

**Reply by artel1992** [reply]
Да и пару примеров использования в компанентах
